ScraperFC/SofaScore.py

```diff
@@ -1,144 +1,454 @@
+        on the SofaScore UI, but the ones that make the query param "fields" of the request.
+
+        Returns:
+            list: statistics for league scraping
+        """
+        fields = [
+            'goals',
+            'yellowCards',
+            'redCards',
+            'groundDuelsWon',
+            'groundDuelsWonPercentage',
+            'aerialDuelsWon',
+            'aerialDuelsWonPercentage',
+            'successfulDribbles',
+            'successfulDribblesPercentage'
+            'tackles',
+            'assists',
+            'accuratePassesPercentage',
+            'totalDuelsWon',
+            'totalDuelsWonPercentage',
+            'minutesPlayed',
+            'wasFouled',
+            'fouls',
+            'dispossessed',
+            'possesionLost',
+            'appearances',
+            'started',
+            'saves',
+            'cleanSheets',
+            'savedShotsFromInsideTheBox',
+            'savedShotsFromOutsideTheBox',
+            'goalsConcededInsideTheBox',
+            'goalsConcededOutsideTheBox',
+            'highClaims',
+            'successfulRunsOut',
+            'punches',
+            'runsOut',
+            'accurateFinalThirdPasses',
+            'bigChancesCreated',
+            'accuratePasses',
+            'keyPasses',
+            'accurateCrosses',
+            'accurateCrossesPercentage',
+            'accurateLongBalls',
+            'accurateLongBallsPercentage',
+            'interceptions',
+            'clearances',
+            'dribbledPast',
+            'bigChancesMissed',
+            'totalShots',
+            'shotsOnTarget',
+            'blockedShots',
+            'goalConversionPercentage',
+            'hitWoodwork',
+            'offsides',
+            'expectedGoals',
+            'errorLeadToGoal',
+            'errorLeadToShot',
+            'passToAssist'
+        ]
+        return fields
+    
+    def get_tournaments_ids(self):
+        """Returns the SofaScore league id (that appears on the URL).
+        Example: https://www.sofascore.com/tournament/football/argentina/copa-de-la-liga-profesional/13475#47644 --> 13475 is the league id
+
+        Returns:
+            dict: 
+                Key: League
+                Value: League id
+        """
+        tournaments_ids = {
+            'La Liga': 8,
+            'Premier League': 17,
+            'Copa de la Liga Profesional': 13475
+        }
+        return tournaments_ids
+
+    def get_seasons(self, year, tournament):
+        """Returns the SofaScore season id (that appears on the URL)
+        Example: https://www.sofascore.com/tournament/football/argentina/copa-de-la-liga-profesional/13475#47644 --> 47644 is the season id
+        Seasons may vary if the leagues start on January or June/July.
+
+        Returns:
+            dict: 
+                Key: Season
+                Value: Season id
+        """
+
+        season_ids = {
```

Comment:
These season IDs are not the same for all leagues. The 23/24 ID for the EPL is not the same as for La Liga. So the tournament will need to be taken into account when selecting the ID. I get an error when I call `scrape_league_stats("La Liga", "23/24")` because of this. `scrape_league_stats("Premier League", "23/24")` works.